src/Tests/Service/U2fServiceAuthenticationVerificationTest.php

```diff
@@ -252,4 +268,96 @@ class_exists('u2flib_server\U2F');
             [235789],
         ];
     }
+
+    /**
+     * @test
+     * @group authentication
+     * @dataProvider expectedDeviceErrors
+     *
+     * @param int $deviceErrorCode
+     */
+    public function it_handles_expected_device_errors($deviceErrorCode)
+    {
+        $keyHandle = 'key-handle';
+        $challenge = 'challenge';
+
+        $request = new \Surfnet\StepupU2fBundle\Dto\SignRequest();
+        $request->version   = \u2flib_server\U2F_VERSION;
+        $request->challenge = $challenge;
+        $request->appId     = self::APP_ID;
+        $request->keyHandle = $keyHandle;
+
+        $response = new \Surfnet\StepupU2fBundle\Dto\SignResponse();
+        $response->errorCode = $deviceErrorCode;
```

Comment:
hehe, humour me, please either align all assignments (as done on lines 285-288, which has my personal preference as it is easier to follow/read/skim with the mass assignments) or align nothing.